packages/amplication-client/src/Plugins/hooks/usePlugins.ts

```diff
@@ -217,6 +220,30 @@ const usePlugins = (resourceId: string, pluginInstallationId?: string) => {
     },
   });
 
+  const [createPluginInstallations] = useMutation<{
+    createPluginInstallations: models.PluginInstallation[];
+  }>(CREATE_PLUGINS_INSTALLATION, {
+    onCompleted: (data) => {
+      data.createPluginInstallations.forEach((plugin) => {
+        addBlock(plugin.id);
+      });
+    },
+    refetchQueries: [
```

Comment:
is it possible to return from API the nee GET_PLUGIN_INSTALLATIONS instead of refetch it ? it will be much more efficient and will solve two issues in one. graphQl any way return the new result, why not combine them together. also, this call should happen only once as part of the migration, so I expect this to be separated and one time in any project

Reply:
Remove the reftech queries